Background Scripts/Refresh Dashboards/script.js

```diff
@@ -0,0 +1,19 @@
+// Define the sys_id of the dashboard you want to refresh
+var dashboardSysId = 'YOUR_DASHBOARD_SYS_ID';
+
+// Function to simulate the dashboard refresh
+function refreshDashboard(dashboardSysId) {
+    // Use the Service Portal widget for dashboards to refresh them
+    var dashboardGR = new GlideRecord('pa_dashboards');
+    if (dashboardGR.get(dashboardSysId)) {
+        // Trigger dashboard refresh by updating a field or simulating a reload
+        dashboardGR.setValue('last_refreshed', gs.nowDateTime());

```

Comment:
This field does not exist on W and X instances. Additionally, ServiceNow dashboards load their data on open and can be refreshed through a page. Reports can be run on a schedule for emails